packer/vagrant-box.json

```diff
@@ -247,12 +247,23 @@
                 "# Install helm",
                 "wget -O - https://kubernetes-helm.storage.googleapis.com/helm-v2.5.1-linux-amd64.tar.gz | tar xz -C /usr/local/bin --no-same-owner --strip-components=1 linux-amd64/helm",
                 "sudo -i -u vagrant helm init",
+                "systemctl stop kube-apiserver kube-controller-manager kube-proxy kube-scheduler kubelet docker",
+                "rm -rf /data/docker/image/* /data/docker/containers/* /data/docker/overlay2",
                 "# Install additional packages",
                 "zypper --non-interactive install jre-openjdk-headless make zip"
             ]
         },
         {
             "type": "shell",
+            "only": ["vmware-iso", "virtualbox-iso", "qemu"],
```

Comment:
Why isn't this just shared between everyone?

Reply:
Because for the Triton build it needs to delete docker stuff in /data, not /var/lib/docker. See above

Reply:
Oh right, we don't do the bind mount. 👍